server/scripts/download_mmctl_release.sh

```diff
@@ -16,10 +16,11 @@ fi
 BIN_PATH=${2:-bin}
 
 
+## If pattern release-X.Y exist in branch name we parse the release branch and we fallback to masterx
 THIS_BRANCH=$(git rev-parse --abbrev-ref HEAD)
 RELEASE_PATTERN='release-[0-9]+\.[0-9]+(\.[0-9]+)?'
 if [[ $THIS_BRANCH =~ $RELEASE_PATTERN ]]; then
-  RELEASE_TO_DOWNLOAD=$THIS_BRANCH
+  RELEASE_TO_DOWNLOAD=$(echo $THIS_BRANCH | grep -Eo 'release-(\d+)?\.(\d+)?')
```

Comment:
The \d wasn't working for me. This worked: `grep -Eo 'release-([0-9]+)?\.([0-9]+)?'`. Can you check?

Reply:
Check with different grep versions across OS and the `\d` solution is not consistent . Switched to the numbered one . 
Nice one 👍🏼